Jenkinsfile

```
@@ -29,7 +29,9 @@ def generateParallelSteps(labels) {
           sh "bash ./checks.sh " + label + " '${env.NODE_NAME}' "
         } else {
           //pwsh (script: ".\\checks.ps1 '${env.NODE_NAME}' ")
+          env
```

`env` is not a pipeline keyword. It is a groovy variable holding an object (with attributes for each environment variables seen in the groovy sandbox on the controller, not on the shell/powershell agent process).

Calling it directly in the pipeline won't work as you expect.


Instead, run it on a an agent shell process:

Either

```suggestion
          pwsg 'dir env:'
```

or

```suggestion
    bat 'set'
```